.travis.yml

```diff
@@ -215,14 +231,23 @@ install:
       export PATH=$PWD/gcc-arm-8.2-2018.11-x86_64-arm-linux-gnueabihf/bin:$PATH;
       arm-linux-gnueabihf-g++ --version;
       ./gcc-arm-8.2-2018.11-x86_64-arm-linux-gnueabihf/bin/arm-linux-gnueabihf-g++ --version;
-      travis_retry sudo apt-get install -y qemu-user-static;
       wget --no-verbose -O ubuntu-base.tar.gz http://cdimage.ubuntu.com/ubuntu-base/releases/19.10/release/ubuntu-base-19.10-base-armhf.tar.gz;
       sudo mkdir rootfs;
       sudo tar xf ubuntu-base.tar.gz -C rootfs;
       sudo cp /usr/bin/qemu-arm-static rootfs/usr/bin/;
       sudo chroot rootfs mount -t proc none /proc;
     fi'
 
+  - 'if [[ "$arch" == "s390x" ]]; then
+      cd $TRAVIS_BUILD_DIR/test;
+      s390x-linux-gnu-g++-8 --version;
+      wget --no-verbose -O ubuntu-base.tar.gz http://cdimage.ubuntu.com/ubuntu-base/releases/19.10/release/ubuntu-base-19.10-base-s390x.tar.gz;
+      sudo mkdir rootfs;
+      sudo tar xf ubuntu-base.tar.gz -C rootfs;
+      sudo cp /usr/bin/qemu-s390x-static rootfs/usr/bin/;
+      sudo chroot rootfs mount -t proc none /proc;
```

Comment:
shouldn't all these commands be separated by `&&` instead of `;`? if one fails, will the error propagate?

Reply:
it's probably a good idea, but if any of them fails, everything else will fail